test/JDBC/expected/rowcount-vu-verify.out

```diff
@@ -56,9 +118,59 @@ int
 ~~END~~
 
 
+
 set rowcount -1
 GO
 ~~ERROR (Code: 33557097)~~
 
 ~~ERROR (Message: -1 is outside the valid range for parameter "babelfishpg_tsql.rowcount" (0 .. 2147483647))~~
 
+
+set rowcount NULL
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: syntax error near 'NULL' at line 1 and character position 13)~~
+
+
+declare @v smallint = NULL
+set rowcount @v
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: null value cannot be assigned to variable "babelfishpg_tsql.rowcount" declared NOT NULL)~~
```

Comment:
SQL Server message is: ` Invalid argument for SET ROWCOUNT. Must be a non-null non-negative integer.`  Do not use Babelfish-specific implementation details like the GUC name in a user-facing error message.